### PR TITLE
Tls support

### PR DESCRIPTION
**Pull request by koiuo**

### CODE REVIEW

File: acctest/stubserver.go [hunk 1]
```diff
@@ -3,11 +3,22 @@ package acctest
 import (
```
Comment (ashald) [inline acctest/stubserver.go:23]: This file still missing public domain notice while it present in other files. :D We want to be uniform so let's add it here or drop it from everywhere except the license file.

File: main.go [hunk 2]
```diff
@@ -87,6 +93,21 @@ func createApp(mainFn mainFn) *cli.App {
...
+			Name:        "tls",
+			Usage:       "Use TLS, verify server with CA certificates installed on this host",
+			Destination: &flags.tls,
+		},
+		cli.BoolFlag{
+			Name:        "tls-insecure",
+			Usage:       "Use TLS, do NOT verify server (accept any certificate)",
+			Destination: &flags.tlsInsecure,
+		},
+		cli.StringFlag{
```
Comment (ashald) [inline main.go:108]: So how about `tls-dir` option so it will be possible to supply several CA certs?

**Review by ashald — COMMENTED** [review]
I just posted few minor questions

File: main.go (hunk 2 above)
Comment (koiuo) [reply main.go:108]: Please see updated code

**Comment by edio** [comment]
I also renamed Test methods to form `TestXxx` where `Xxx` is alphanumeric with first `X` is not `[a-z]` as  [documentation](https://golang.org/pkg/testing/) requires

File: acctest/stubserver.go (hunk 1 above)
Comment (koiuo) [reply acctest/stubserver.go:23]: fixed

**Review by ashald — APPROVED** [review]
LGTM!

**Pull request merged**